src/main/webapp/directives/d3Annotation.js

```diff
@@ -5,14 +5,14 @@
  */
```

Comment:
There are a lot of "Rounded values increase performance" comments. You can remove them and insert this comment instead one time in the header of this file.